### PR TITLE
duck 6.0.1

### PR DESCRIPTION
**Pull request by dkocher**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/duck.rb [hunk 1]
```diff
@@ -2,8 +2,8 @@ class Duck < Formula
   desc "Command-line interface for Cyberduck (a multi-protocol file transfer tool)"
   homepage "https://duck.sh/"
   # check the changelog for the latest stable version: https://cyberduck.io/changelog/
-  url "https://dist.duck.sh/duck-src-6.0.0.24625.tar.gz"
-  sha256 "ea9ac134fa3146822fced3da34f56154616993008707e74e8fc0733e7617425e"
+  url "https://dist.duck.sh//duck-src-6.0.1.24918.tar.gz"
```
Comment (dunn) [inline Formula/duck.rb]: You've got an extra slash in here.

Comment (dkocher) [inline Formula/duck.rb]: Not an issue. The URI is valid.

Comment (MikeMcQuaid) [inline Formula/duck.rb]: It works without the slash so please remove it, thanks.

**Comment by ilovezfs** [comment]
Superseded by https://github.com/Homebrew/homebrew-core/pull/14270.

@dkocher but thanks for the PR nonetheless

**Pull request closed without merging**